Google.PowerShell.IntegrationTests/BigQuery/BigQuery.BqJob.Tests.ps1

```diff
@@ -135,4 +135,33 @@ Describe "BqJob-Query" {
     }
 }
 
+Describe "Stop-BqJob" {
+
+    BeforeAll {
+        $r = Get-Random
+        $datasetName = "pshell_testing_$r"
+        $test_set = New-BqDataset $datasetName
+        $folder = Get-Location
+        $folder = $folder.ToString()
+        $filename = "$folder\classics_large.csv"
+        $table = New-BqTable -Dataset $test_Set "table_$r"
+        New-BqSchema -Name "Title" -Type "STRING" | New-BqSchema -Name "Author" -Type "STRING" |
+            New-BqSchema -Name "Year" -Type "INTEGER" | Set-BqSchema $table | 
+            Add-BqTabledata $filename CSV -SkipLeadingRows 1
+    }
+
+    It "should stop a query job" {
+        $job = Start-BqJob -Query "select * from book_data.classics where Year > 1900"
```

Comment:
What happened if you call Stop-BqJob on a job that is already stopped. Can we add a test case for that?